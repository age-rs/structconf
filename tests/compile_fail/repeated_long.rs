//! Shouldn't compile because an identifier is repeated

use structconf::StructConf;

#[derive(StructConf)]
struct Config {
    #[conf(no_file, no_short, long = "abc")]
    pub name1: bool,
    #[conf(no_file, no_short, long = "abc")]
    pub name2: bool,
}

fn main() {}
