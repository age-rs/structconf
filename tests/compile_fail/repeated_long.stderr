error: The `long` attribute with value `abc` is repeated with another field.
  --> $DIR/repeated_long.rs:10:9
   |
10 |     pub name2: bool,
   |         ^^^^^
